### ISSUE TITLE
Network interface configuration

### ISSUE THREAD

**Issue by ricardomv** [papyros/qml-desktop#5]
Started working on qml interface for listing network devices and establish a connection

uses https://projects.kde.org/projects/frameworks/networkmanager-qt

**Comment by boghison** [comment]
Awesome! Don't forget to use QQuickList for animations ;)

**Comment by ricardomv** [comment]
yes i will base it on the list model class

**Comment by eyelash** [comment]
The KDE Plasma widget for network management uses QML as well as far as I know, so somewhere there must exist QML bindings. Have you considered using the bindings that KDE uses?

(from a quick search it seems to me that [these](http://quickgit.kde.org/?p=plasma-nm.git&a=tree&f=libs%2Fdeclarative) are their bindings)

**Comment by ricardomv** [comment]
@eyelash yes that's what i have been prototyping with but some of the api does not work for us. For example it provides a connection icon for a wireless network in a string format ("network-wireless-20-locked") the problem is that we need 6 icons (0,20,40,60,80,100) to make it work but we only have 5. Also another concern are the dependencies.
